sklearn/metrics/classification.py

```diff
@@ -1035,8 +1059,13 @@ def precision_score(y_true, y_pred, labels=None, pos_label=1,
     y_pred : 1d array-like, or label indicator array / sparse matrix
         Estimated targets as returned by a classifier.
 
-    labels : array
-        Integer array of labels.
+    labels : list, optional
+        The set of labels to include when ``average != 'binary'``, and their
+        order if ``average is None``. Labels present in the data can be
+        excluded, for example to calculate a multiclass average ignoring a
+        majority negative class, while labels not present in the data will
+        result in 0 components in a macro average.  By default, all labels in
+        ``y_true`` and ``y_pred`` are used in sorted order.
```

Comment:
Could you add a comment about the `labels` in multi-label classification?

Otherwise thanks, the docstring is a lot better!